### PR TITLE
Included further clarity around GC + incremental error

### PR DESCRIPTION
**Pull request by kathancox**
Fixes DOC-4966, DOC-6130

Added further definition around avoiding the `protected is verification error`. Also, for v22.2, added a ref to the scheduled backups + protected timestamp decoupling from GC to the common errors section.

## Preview

https://deploy-preview-15806--cockroachdb-docs.netlify.app/docs/stable/common-errors.html#protected-ts-verification-error

### CODE REVIEW

**Review by msbutler — APPROVED** [review]
Looks great. Thanks for doing this!

**Review by andf-crl — APPROVED** [review]
LGTM with one optional suggestion. Thanks!

File: v22.1/common-errors.md [hunk 1]
```diff
@@ -149,7 +149,7 @@ The error message will specify which part of your backup is causing the failure.
 
 To resolve this issue, take a new [full backup](take-full-and-incremental-backups.html) after doing either of the following:
 
-- Increase the garbage collection period by [configuring the `gc.ttlseconds` replication zone variable](configure-replication-zones.html#gc-ttlseconds), or
+- Increase the garbage collection period by [configuring the `gc.ttlseconds` replication zone variable](configure-replication-zones.html#gc-ttlseconds). For example, we recommend setting the GC TTL to a time interval **greater** than `incremental_backup_interval` + `expected_runtime_of_full_backup` + `buffer_for_slowdowns`. To estimate the expected full backup runtime, it is necessary to perform testing or verify the past performance through the [jobs table](ui-jobs-page.html#jobs-list).
```
[the remark this answers is not in the document]
Comment (kathancox) [reply v22.1/common-errors.md]: Good call out! Done ✅

**Comment by kathancox** [comment]
TFTRs!

**Pull request merged**